packages/appChrome/components/HeaderBar.tsx

```diff
@@ -1,15 +1,68 @@
 import * as React from "react";
 import { cx } from "emotion";
-import { darkMode, flex, padding } from "../../shared/styles/styleUtils";
+import {
+  background,
+  darkMode,
+  flex,
+  padding
+} from "../../shared/styles/styleUtils";
 import { headerBar, appChromeInsetContent } from "../style";
 
 export interface HeaderProps {
+  backgroundColor?: string;
   children: React.ReactElement<HTMLElement> | string;
+  paddingBottom?:
```

Comment:
Types are like variables for structure, we could easily make this shorter and more readable.

```suggestion
  paddingBottom?: SizingOption
```


Somewhere else (ideally in a place for base types):

```ts
export type SizingOption = 
  | "xxs"
  | "xs"
...
```

Reply:
I don't know where the place for base types is, which is why I asked.

Reply:
For example, `display` here doesn't have a place for base types - https://github.com/dcos-labs/ui-kit/blob/90280672f59c9af244a8c2e5838294c3bb5969a7/packages/shared/styles/styleUtils/modifiers/display.ts#L3 .

Reply:
There is none yet, for example `packages/types`

Reply:
I would prefer to wait for @mperrotti to discuss the best place to have these, since he is the one who does most of the work here.

Reply:
@DanielMSchmidt - I like the idea of having a place for types like `packages/types`